src/dotnet-ildasm.Tests/Infrastructure/TypeFormatExtensionsShould.cs

```diff
@@ -48,7 +48,7 @@ public void Return_IL_Type_For_Int32()
 
             var actual = methodDefinition.Parameters.First().ParameterType.ToILType();
 
-            Assert.Equal("int", actual);
+            Assert.Equal("int32", actual);
```

Comment:
Does IL support both _int_ and _int32_ as valid counterparts for System.Int32? Have you run this against MS ILasm?